app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,281 @@
+<p>
+    Appwrite provides authentication for many different use cases to fit the needs of developers. 
+    Appwrite manages authentication with a combination of accounts and sessions. 
+    Accounts can be created in many different ways, such as through an anonymous session, email and password, OAuth authentication, magic URLs, and more.
+</p>
+
+<h2><a href="/docs/authentication#account-vs-user-service" id="account-vs-user-service">Account vs User Service</a></h2>
+
+<p>
+    The Account API operates in the scope of the currently logged-in account and usually used in a frontend or mobile app. The Users API is used in backend integrations and operates in an admin scope using an API key with access to all your project users.
+</p>
+
+<p> 
+    Some of the Account API methods are available from Server SDKs when you authenticate with a JWT. This allows you to perform actions on behalf of a project account's user on the backend.
+</p>
+
+<h2><a href="/docs/authentication#create-an-account" id="create-an-account">Create An Account</a></h2>
+
+<p>
+    A user account in Appwrite is the primary way to access information for a given project. Accounts can be created in many different ways, including <a href="/docs/client/account#accountCreate">email & password</a>, <a href="/docs/client/account#accountCreateAnonymousSession">anonymous sessions</a>, <a href="/docs/client/account#accountCreateOAuth2Session">OAuth2</a>, <a href="/docs/client/account#accountCreatePhoneSession">phone authentication</a>, and more. Applications can create and manage authentication sessions through the REST API or Client SDKs.
+</p>
+
+<h3><a href="/docs/authentication#email" id="email">Email</a></h3>
+
+<p>
+    Creating an account via email and password is one of the most common ways to sign up for an application. Appwrite provides email and password authentication out of the box. Using one of Appwrite's Client SDKs, or the REST APIs directly, you can create an account using an email address and password in your application.
```

Comment:
We could also mention that passwords are hashed with Argon2, a secure password hashing algo.